pyanaconda/kickstart.py

```diff
@@ -1218,6 +1218,10 @@ class Nvdimm(commands.nvdimm.RHEL7_Nvdimm):
     def parse(self, args):
         action = commands.nvdimm.RHEL7_Nvdimm.parse(self, args)
 
+        if not blivet.nvdimm.nvdimm().nvdimm_plugin_available:
```

Comment:
I think that `blivet.nvdimm.nvdimm` is already the instance of NVDIM you want, so why do you call `nvdimm()`?

Reply:
It is just consistent with already present nvdimm singleton calls, and also with how we are already using eg iscsi, fcoe and ZFCP singleton. Which I hesitate to change in scope of this PR.

Reply:
No problem, I was just curious.